fifteen_min/src/viewer.rs

```diff
@@ -18,6 +18,8 @@ use widgetry::{
     Line, Outcome, Panel, RewriteColor, State, Text, Toggle, Transition, VerticalAlignment, Widget,
 };
 
+use std::str::FromStr;
```

Comment:
Style nit: Place `std` imports in their own block before the `abstuil` one here. Unfortunately `cargo fmt` doesn't group imports, so we do this manually. We switched to this "multiple paragraph" style a while ago because somebody was contributing and using an IDE that insisted on this sorting. The order is stuff from `std`, then external crates, then crates within the abstreet repo, then stuff from within the current crate.

Reply:
Make sense and yea I am using the Jetbrains IDE where I think they created their own analyzer so its probably just an issue there. I can compile the crate as well so I will ignore it, just wanted to check on it.